slices/base-files.yaml

```diff
@@ -28,7 +28,7 @@ slices:
 
   tmp:
     contents:
-      /tmp/:
+      /tmp/: { mode: 01777 }
```

Comment:
This shouldn't be required, as it's the actual mode of the underlying directory. I think we have a bug about umask getting in the way during creation. Is that what you're seeing? Why was the explicit mode added here?

Reply:
I though there wasn't bug but that the directory had wrong permissions in the archive since the postinst script explicitly adjusted its permissions. But I've just checked it's correct and indeed it sounds like a bug with umask handling. Reported here: https://github.com/canonical/chisel/issues/9